packages/account/src/Sections/Profile/FinancialAssessment/financial-assessment.jsx

```diff
@@ -222,6 +223,43 @@ class FinancialAssessment extends React.Component {
                 this.props.removeNotificationByKey({ key: 'risk' });
             }
             setSubmitting(false);
+            const {
+                income_source,
+                employment_status,
+                employment_industry,
+                occupation,
+                source_of_wealth,
+                education_level,
+                net_income,
+                estimated_worth,
+                account_turnover,
+                binary_options_trading_frequency,
+                cfd_trading_experience,
+                cfd_trading_frequency,
+                forex_trading_experience,
+                forex_trading_frequency,
+                other_instruments_trading_experience,
+                other_instruments_trading_frequency,
+            } = values;
+
+            this.setState({
+                income_source,
+                employment_status,
+                employment_industry,
+                occupation,
+                source_of_wealth,
+                education_level,
+                net_income,
+                estimated_worth,
+                account_turnover,
+                binary_options_trading_frequency,
+                cfd_trading_experience,
+                cfd_trading_frequency,
+                forex_trading_experience,
+                forex_trading_frequency,
+                other_instruments_trading_experience,
+                other_instruments_trading_frequency,
+            });
```

Comment:
(question)(non-blocker) Why do we need to set the state here?

Reply:
This updates the states to the values specified by the client when the form is submitted with new values, which will in turn trigger the reinitialisation of Formik.